tox.ini

```diff
@@ -55,6 +55,7 @@ deps =
         minio
         django-stubs==4.2.*
         Django==4.2.*
+        types-requests
```

Comment:
This isn't necessary to fix any errors, but it will improve type checking coverage.